server/src/services/taskResults.service.ts

```diff
@@ -198,11 +198,13 @@ export async function getTaskSolutionFeedback(studentId: number, courseTaskId: n
           discord: c.checker.user.discord,
         }
       : null;
+    const criteriaData = c.historicalScores.sort((a, b) => b.dateTime - a.dateTime);
     return {
       author,
       updatedDate: c.updatedDate,
       comment: c.comment,
       score: c.score,
+      criteria: criteriaData[0].criteria,
```

Comment:
why not use destructuring here?
const [{criteria}] = c.historicalScores.sort((a, b) => b.dateTime - a.dateTime);

Reply:
nice remark. I've got it